exampleCourse/questions/demo/drawing/buttons/question.html

```
@@ -3,12 +3,18 @@
             <pl-controls-group label="Misc">
-                <pl-drawing-button type="pl-coordinates"></pl-drawing-button>
+                <pl-drawing-button type="pl-coordinates" label="O" angle="0" width="60" x1="140" y1="140" relpos-active="true"></pl-drawing-button>
                 <pl-drawing-button type="pl-line"></pl-drawing-button>
                 <pl-drawing-button type="pl-arc"></pl-drawing-button>
-                <pl-drawing-button type="pl-point"></pl-drawing-button>
+                <pl-drawing-button type="pl-point" label="A"></pl-drawing-button>
                 <pl-drawing-button type="pl-text"></pl-drawing-button>
             </pl-controls-group>
+            <pl-controls-group label="Forces and Vectors">
+                <pl-drawing-button type="pl-vector" label="\vec{v}" label-latex="true" x1="50" y1="100" angle="-40" label-relpos="ahead" offsety="100"></pl-drawing-button>
+                <pl-drawing-button type="pl-double-headed-vector" label="T" label-relpos="ahead"x1="50" y1="100"></pl-drawing-button>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </pl-drawing-button> ]

(tag-pair)